ex/app/src/main/java/com/example/ex/MainFragment.java

```diff
@@ -0,0 +1,209 @@
+package com.example.ex;
+
+import android.content.Context;
+import android.os.Bundle;
+import androidx.annotation.NonNull;
```

Comment:
Some panel with shadow is visible at the corner of the collapsed toolbar 
![device-scrn-2020-11-12-00-59-42](https://user-images.githubusercontent.com/4456745/98869239-84f61400-2482-11eb-94a1-bef04e4b1945.png)